docs/rackhd/install_os.rst

```diff
@@ -107,6 +108,18 @@ ipv6           Object   *optional*    See `ipv4 or ipv6`_ more details.
 esxSwitchName  String   *optional*    **(ESXi only)** The vswitch to attach the vmk device to. vSwitch0 is used by default if no esxSwitchName is specified.
 ============== ======== ============ ============================================
 
+.. _installPartitions:
+
+For **installPartitions** in payload:
+
+============== ======== ============ ============================================
+Parameters     Type     Flags        Description
+============== ======== ============ ============================================
+mountPoint     String   **required**  Mount point, it could be "/boot", "/", "swap", etc. just like the mount point input when manually installing OS.
+size           String   **required**  Partition size, it could be a number string or "auto", For number, default unit is **MB**. For "auto", all available free disk space will be used.
```

Comment:
`size` is a `number` will be more straightforward. If size is not specified,  then you can default to "auto"